api/getAllContactsByUserId.php

```diff
@@ -0,0 +1,63 @@
+<?php
+// Load environment variables
+require_once 'vendor/autoload.php';
+$dotenv = Dotenv\Dotenv::createImmutable(__DIR__);
+$dotenv->load();
+
```

Comment:
where is the http header?

Reply:
deleted them on accident should be fixed now